.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "pwa-chrome",
+      "request": "launch",
+      "name": "Launch Chrome against localhost",
+      "url": "http://localhost:8080",
+      "webRoot": "${workspaceFolder}"
+    }
+  ]
+}
```

Comment:
Was this file intended to be checked in?

Reply:
I wondered as well — did also notice that we have this VS Code debugger helper [in various projects](https://github.com/search?q=org%3Aartsy+path%3A.vscode&type=Code)

Reply:
Iif there's a launch config that's helpful to the wider team we should announce!

Reply:
Yeah, actually this was a mistake 😅 - I spent a while configuring things so I could debug the tests within VSCode, but I didn't end up using this Chrome profile anyway, so I'll just quietly remove it...